sdk/src/loom/Graphics/Shader.ls

```diff
@@ -0,0 +1,171 @@
+/*
+===========================================================================
+Loom SDK
+Copyright 2011, 2012, 2013
+The Game Engine Company, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+===========================================================================
+*/
+
+package loom.graphics
+{
+    import loom2d.math.Matrix;
+    import loom2d.textures.Texture;
+
+    delegate ShaderBindDelegate();
+
+    /** Shader needs to be initialized after being created with 'load()' or 'loadFromAssets()'.
```

Comment:
You should summarize this. ie, "Shader allows custom GLSL shaders to be used when drawing Quads and QuadBatches."